documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -2,15 +2,17 @@
 
 ## Version 9.3.0
 
-### New features
-
 ### Changed features
 
 * Any arguments that specify the number of threads to be used provided as part of the `detect.maven.build.command` [solution_name] property will be omitted when executing the Maven CLI.
 
 ### Resolved Issues
 
+* (IDETECT-4164) Component Location Analysis feature has improved parser support for package managers like Poetry that employ variable delimiters for better location accuracy.
```

Comment:
Non-critical rewording suggestion: * (IDETECT-4164) Improved Component Location Analysis parser support for package managers like Poetry that employ variable delimiters, for better location accuracy.